apply/tasks.py

```diff
@@ -20,7 +21,9 @@ def send_application_deadline_reminders_impl() -> None:
 
     rounds_for_reminders = get_rounds_for_application_deadline_reminders()
     for round in rounds_for_reminders:
-        incomplete_applications = get_unsubmitted_applications_for_round(round.id)
+        incomplete_applications = get_applications_for_round_by_status(
+            round.id, [Status.NOT_STARTED, Status.IN_PROGRESS, Status.COMPLETED]
```

Comment:
I can see this comes directly from the method it used to call but I'm guessing there's a sensible reason something called `incomplete_applications` filters for applications in the `Status.COMPLETED` state? 

(This is a refactor and has nothing to do with this change so not this pull requests question)

Reply:
Haha.. I asked @ksp37-dluhc the same question, and he wondered the same.

There is a concept of completed application, but un-submitted. 

From Assess' POV, that's not "completed". So we notify the users with to submit their application before the round ends.

The variable name is misleading.